### ISSUE TITLE
scrollToTop after Find-in-Page expands amp-accordion section

### ISSUE THREAD

**Issue by caroqliu** [ampproject/amphtml#28076]
## What's the issue?

Note: This issue is filed regarding an experimental feature. 

The first time an `amp-accordion > section` expands on `rendersubtreeactivation` event activation, the page scrolls the user position to the top of the page. The second time that the user will `Cmd/Ctrl+F` back to it, they get scrolled to the appropriate section where the text is located in the newly-expanded section on the page. 

Related: [crbug.com/1075997](https://crbug.com/1075997)

## How do we reproduce the issue?

1. Go to [this product page](https://www-vova-com.cdn.ampproject.org/c/s/www.vova.com/amp/Easy-Breath-Repiration-Mask-With-Valve-Pm2-5-Protection-Mak-With-Mask-SN1585381085467277258044386-g17048886-m18515936)
2. Open the console and toggle the AMP experiment on with `AMP.toggleExperiment('amp-accordion-display-locking', true)` and refresh.
3. `Cmd/Ctrl + F` with text "Don't worry" -- when section expands, the user will be scrolled to the top. 
4. Continue pressing `Cmd/Ctrl + F` on the same text, it will eventually bring you to the appropriate location.

Expected behavior: `3` does not occur. Specifically, when the section expands, the user should be scrolled to the appropriate location where the target text is located on the document.

## What browsers are affected?

Chrome 81

**Comment by caroqliu** [comment]
To fix on the AMP side: Modify the `rendersubtreeactivation` event handler to include scrolling to the target section after expand.

**Comment by caroqliu** [comment]
Closing this issue as we are moving to [the CSS property approach](https://github.com/WICG/display-locking/blob/master/README.md)
